### PR TITLE
lca: Limiting the client for pull and newbuilder for lca

### PR DESCRIPTION
**Pull request by achuzhoy**
This is based on assignment
https://github.com/openshift-kni/eco-goinfra/issues/299

### CODE REVIEW

**Comment by achuzhoy** [comment]
cc @trewest @kononovn

**Comment by achuzhoy** [comment]
successfully tested

**Review by sebrandon1 — APPROVED** [review]

**Comment by sebrandon1** [comment]
It's interesting that the `Client` [variable](https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/client/interfaces.go#L164) is actually an interface.

**Comment by trewest** [comment]
> It's interesting that the Client [variable](https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/client/interfaces.go#L164) is actually an interface.

@sebrandon1 I think [this](https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/client/client.go#L223) is the underlying struct that implements Client. It's set [here](https://github.com/openshift-kni/eco-goinfra/blob/main/pkg/clients/clients.go#L181)

**Review by trewest — CHANGES_REQUESTED** [review]
Just one minor comment

File: pkg/lca/seedgenerator.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/openshift-kni/eco-goinfra/pkg/msg"
 	lcasgv1alpha1 "github.com/openshift-kni/lifecycle-agent/api/seedgenerator/v1alpha1"
 	k8serrors "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
[the remark this answers is not in the document]
Comment (achuzhoy) [reply pkg/lca/seedgenerator.go]: Updated. Thanks.

**Review by trewest — APPROVED** [review]
/lgtm

**Pull request merged**